ehsm_kms_service/secret_management.js

```diff
@@ -0,0 +1,64 @@
+const crypto = require('crypto')
+const logger = require('./logger')
+const {
+    CMK_EFFECTIVE_DURATION,
+    KEYID_SYSTEM_DEFAULT_CMK
+} = require('./constant')
+const {
+    napi_result,
+    _result,
+} = require('./function')
+const {
+    cryptographic_apis,
+} = require('./apis')
+const ehsm_napi = require('./ehsm_napi')
+
+//initialize Default CMK
+const init_DefaultCMK = (DB) => {
+    try {
+        const keyid = KEYID_SYSTEM_DEFAULT_CMK
+        const query = {
+            selector: {
+                keyid,
+            },
+            fields: ['keyid',],
+            limit: 1,
+        }
+        DB.partitionedFind('cmk', query)
```

Comment:
can reuse store_cmk() ?

Reply:
Parameters and return values are different, if reused，store_cmk() needs to be modified

Reply:
OK.